### PR TITLE
Layout blocks

### PR DESCRIPTION
**Pull request by george-misan**

### CODE REVIEW

File: services/course-material/src/components/ContentRenderer/index.tsx [hunk 1]
```diff
@@ -8,8 +8,11 @@ import ButtonBlock from "./ButtonBlock"
 import CodeBlock from "./CodeBlock"
 import QuoteBlock from "./QuoteBlock"
 import AudioBlock from "./AudioBlock"
+<<<<<<< HEAD
+=======
```
Comment (nygrenh) [inline services/course-material/src/components/ContentRenderer/index.tsx:12]: merge conflict

File: shared-module/package.json [hunk 2]
```diff
@@ -33,6 +33,13 @@
...
     "@emotion/react": "^11.4.0",
     "@emotion/styled": "^11.3.0",
-    "react-query": "^3.16.1"
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/react-fontawesome": "^0.1.14",
+    "axios": "^0.21.1",
+    "react-animated-burgers": "^1.2.8",
+    "react-query": "^3.16.1",
+    "styled-components": "^5.3.0",
```
Comment (nygrenh) [inline shared-module/package.json:42]: no styled components

**Review by nygrenh — CHANGES_REQUESTED** [review]

File: services/course-material/src/components/ContentRenderer/AudioBlock.tsx [hunk 3]
```diff
@@ -25,4 +26,4 @@ const AudioBlock: React.FC<BlockRendererProps<AudioAttributes>> = ({ data }) =>
   )
 }
 
-export default withErrorBoundary(AudioBlock)
+export default AudioBlock
```
Comment (nygrenh) [inline services/course-material/src/components/ContentRenderer/AudioBlock.tsx:29]:
```suggestion
export default withErrorBoundary(AudioBlock)
```

File: services/course-material/src/components/ContentRenderer/AudioBlock.tsx [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import { css } from "@emotion/css"
 
-import withErrorBoundary from "../../shared-module/utils/withErrorBoundary"
 import { normalWidthCenteredComponentStyles } from "../../styles/componentStyles"
```
Comment (nygrenh) [inline services/course-material/src/components/ContentRenderer/AudioBlock.tsx:3]:
```suggestion
import withErrorBoundary from "../../shared-module/utils/withErrorBoundary"
import { normalWidthCenteredComponentStyles } from "../../styles/componentStyles"
```

**Review by nygrenh — CHANGES_REQUESTED** [review]
+ merge conflict in shared-module

**Review by jolampi — APPROVED** [review]

**Pull request merged**